### ISSUE TITLE
Bug: v4.2.9 version number is not updated

### ISSUE THREAD

**Issue by prbt2016** [codeigniter4/CodeIgniter4#6797]
### PHP Version

7.4

### CodeIgniter4 Version

4.2.9

### CodeIgniter4 Installation Method

Composer (using `codeigniter4/appstarter`)

### Which operating systems have you tested for this bug?

Linux

### Which server did you use?

apache

### Database

MySQL 5.5

### What happened?

Hello,

Was in the process of installation of Codeigniter 4.2.9 using composer on Centos 7 with Apache 2.2, PHP 7.4/8.0  i.e : 

https://codeigniter.com/user_guide/installation/installing_composer.html#

However after running composer create-project codeigniter4/appstarter  /{{PATH}}/{{TO}}/{{CODEIGNITER-DIR}}/  following is shown i.e :

Installing codeigniter4/appstarter (v4.2.9)

However after visiting the URL i.e :  

http://{{DOMAIN.COM}}/{{CODEIGNITER-DIR}}/public/

The version shown is  4.2.8 and not 4.2.9 as shown in the screenshot below : 

![image](https://user-images.githubusercontent.com/17871448/198962723-ef5047e9-a24c-4eaf-a154-56252e2f1aec.png)

Seems that the version constant isn't updated in 
/{{PATH}}/{{TO}}/{{CODEIGNITER-DIR}}/vendor/codeigniter4/framework/system/CodeIgniter.php

public const CI_VERSION = '4.2.8';

It's 4.2.8 and not 4.2.9 .  Could you please update the same?.

Also while installing on PHP 7.4 dependencies fail to install , i.e after running 
composer create-project codeigniter4/appstarter  /{{PATH}}/{{TO}}/{{CODEIGNITER-DIR}}/ on PHP 7.4 following error is thrown i.e : 

```
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - phpunit/phpunit 9.2.6 requires phpunit/php-code-coverage ^8.0.2 -> satisfiable by phpunit/php-code-coverage[8.0.2].
    - phpunit/phpunit[9.1.0, ..., 9.2.5] require phpunit/php-code-coverage ^8.0.1 -> satisfiable by phpunit/php-code-coverage[8.0.1, 8.0.2].
    - phpunit/phpunit[9.5.16, ..., 9.5.26] require phpunit/php-code-coverage ^9.2.13 -> satisfiable by phpunit/php-code-coverage[9.2.13, ..., 9.2.18].
    - phpunit/phpunit[9.5.10, ..., 9.5.14] require phpunit/php-code-coverage ^9.2.7 -> satisfiable by phpunit/php-code-coverage[9.2.7, ..., 9.2.18].
    - phpunit/phpunit[9.5.0, ..., 9.5.9] require phpunit/php-code-coverage ^9.2.3 -> satisfiable by phpunit/php-code-coverage[9.2.3, ..., 9.2.18].
    - phpunit/phpunit[9.4.0, ..., 9.4.4] require phpunit/php-code-coverage ^9.2 -> satisfiable by phpunit/php-code-coverage[9.2.0, ..., 9.2.18].
    - phpunit/phpunit 9.3.11 requires phpunit/php-code-coverage ^9.1.11 -> satisfiable by phpunit/php-code-coverage[9.1.11, ..., 9.2.18].
    - phpunit/phpunit[9.3.8, ..., 9.3.10] require phpunit/php-code-coverage ^9.1.5 -> satisfiable by phpunit/php-code-coverage[9.1.5, ..., 9.2.18].
    - phpunit/phpunit[9.3.4, ..., 9.3.7] require phpunit/php-code-coverage ^9.1.1 -> satisfiable by phpunit/php-code-coverage[9.1.1, ..., 9.2.18].
    - phpunit/phpunit[9.3.0, ..., 9.3.3] require phpunit/php-code-coverage ^9.0 -> satisfiable by phpunit/php-code-coverage[9.0.0, ..., 9.2.18].
    - phpunit/php-code-coverage[8.0.1, ..., 8.0.2] require theseer/tokenizer ^1.1.3 -> satisfiable by theseer/tokenizer[1.1.3, 1.2.0, 1.2.1].
    - phpunit/php-code-coverage[9.2.16, ..., 9.2.18] require nikic/php-parser ^4.14 -> satisfiable by nikic/php-parser[v4.14.0, v4.15.0, v4.15.1].
    - phpunit/php-code-coverage[9.2.8, ..., 9.2.15] require nikic/php-parser ^4.13.0 -> satisfiable by nikic/php-parser[v4.13.0, ..., v4.15.1].
    - phpunit/php-code-coverage 9.2.7 requires nikic/php-parser ^4.12.0 -> satisfiable by nikic/php-parser[v4.12.0, ..., v4.15.1].
    - phpunit/php-code-coverage[9.2.3, ..., 9.2.6] require nikic/php-parser ^4.10.2 -> satisfiable by nikic/php-parser[v4.10.2, ..., v4.15.1].
    - phpunit/php-code-coverage[9.1.3, ..., 9.2.2] require nikic/php-parser ^4.8 -> satisfiable by nikic/php-parser[v4.8.0, ..., v4.15.1].
    - phpunit/php-code-coverage[9.0.0, ..., 9.1.2] require nikic/php-parser ^4.7 -> satisfiable by nikic/php-parser[v4.7.0, ..., v4.15.1].
    - theseer/tokenizer[1.1.3, ..., 1.2.1] require ext-tokenizer * -> it is missing from your system. Install or enable PHP's tokenizer extension.
    - nikic/php-parser[v4.6.0, ..., v4.15.1] require ext-tokenizer * -> it is missing from your system. Install or enable PHP's tokenizer extension.
    - Root composer.json requires phpunit/phpunit ^9.1 -> satisfiable by phpunit/phpunit[9.1.0, ..., 9.5.26].


```

But dependencies get installed properly when I checked with PHP 8.0 .

What's the cause of the issue ?. Has PHP requirements changed ?. 

Official site shows 7.4 or above : 

https://codeigniter.com/user_guide/intro/requirements.html

Please fix the above issues.

### Steps to Reproduce

1. Run composer create-project codeigniter4/appstarter project-root on PHP 7.4 (Dependencies fail to install with PHP 7.4)
2. Run on PHP 8.0 the same command , CI gets installed
3. Hit installation URL . version shown is incorrect , its 4.2.8 and not 4.2.9.

### Expected Output

1. Dependencies should have been installed on PHP 7.4.
2. Version should have been 4.2.9.

**Comment by kenjis** [comment]
Please enable **ext-tokenizer** in your PHP 7.4.

**Comment by kenjis** [comment]
> Version should have been 4.2.9.

Yes, it is a bug.

**Comment by prbt2016** [comment]
Hello @kenjis ,

> `Please enable ext-tokenizer in your PHP 7.4.`

Thanks . Dependencies do get installed after enabling tokenizer on PHP 7.4 . 

> Yes, it is a bug.

Thanks for acknowledging the same . Could you please update the same and let me know.

**Comment by Terrorboy** [comment]
https://github.com/codeigniter4/CodeIgniter4/pull/6795#issuecomment-1296579069

**Comment by kenjis** [comment]
Closed by v4.2.10 release.

**Comment by prbt2016** [comment]
Hello @kenjis, 

Thanks a lot for the version and version fix . Checked with 4.2.10 and it shows version correctly .

Regards.
